backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/docOutbound/orderDocOutbound.feature

```diff
@@ -0,0 +1,159 @@
+      | pl_so      | ps_1                          | DE                        | EUR                 | price_list_so | true  | false         | 2              |
+    And metasfresh contains M_PriceList_Versions
+      | Identifier | M_PriceList_ID.Identifier | Name   | ValidFrom  |
+      | plv_so     | pl_so                     | plv_so | 2022-01-01 |
+    And metasfresh contains M_ProductPrices
+      | Identifier | M_PriceList_Version_ID.Identifier | M_Product_ID.Identifier | PriceStd | C_TaxCategory_ID.InternalName |
+      | pp_product | plv_so                            | addr_test_product       | 10.0     | Normal                        |
+    And metasfresh contains C_BPartners:
+      | Identifier    | Name          | M_PricingSystem_ID.Identifier | OPT.IsCustomer |
+      | sale_bpartner | sale_bpartner | ps_1                          | Y              |
+
+  Scenario: Create sales order and validate email from doc outbound log - from order disposition
+    Given metasfresh contains C_BPartner_Locations:
+      | Identifier | GLN           | C_BPartner_ID.Identifier | OPT.IsShipTo | OPT.IsBillTo | OPT.EMail          |
+      | bpLocation | 1111123456789 | sale_bpartner            | true         | true         | location@email.com |
+    And metasfresh contains AD_Users:
+      | AD_User_ID.Identifier | Name   | OPT.EMail      | OPT.C_BPartner_ID.Identifier | OPT.C_BPartner_Location_ID.Identifier |
+      | bpUser                | bpUser | user@email.com | sale_bpartner                | bpLocation                            |
+      | bpUser                | bpUser | user@email.com | sale_bpartner                | bpLocation                            |
+    And metasfresh contains S_ExternalReference:
+      | S_ExternalReference_ID.Identifier | ExternalSystem | Type   | ExternalReference | OPT.AD_User_ID.Identifier |
+      | bpUser_ref                        | Shopware6      | UserID | bpUser_ref        | bpUser                    |
+
+    When a 'POST' request with the below payload is sent to the metasfresh REST-API 'api/v2/orders/sales/candidates' and fulfills with '201' status code
+  """
+{
+    "orgCode": "001",
+    "externalHeaderId": 1128101,
+    "externalLineId": 1111,
+    "dataSource": "int-Shopware",
+    "bpartner": {
+        "bpartnerIdentifier": "gln-1111123456789",
+        "bpartnerLocationIdentifier": "gln-1111123456789",
+        "contactIdentifier": "ext-Shopware6-bpUser_ref"
+    },
+    "dateRequired": "2022-02-10",
+    "dateOrdered": "2022-02-02",
+    "orderDocType": "SalesOrder",
+    "paymentTerm": "val-1000002",
+    "productIdentifier": "val-addr_test_product",
+    "qty": 1,
+    "currencyCode": "EUR",
+    "discount": 0,
+    "poReference": "olCand_ref_1128101",
+    "deliveryViaRule": "S",
+    "deliveryRule": "F"
+}
+"""
+
+    And update C_BPartner_Location:
```

Comment:
what is this about? feel free to write a comment right here in case it doesn't fit well into the scenario description (which i guess it doesn't..)

Reply:
to highlight that the email set from from `C_BPartner_Location.EMail` on olCand when is was created is the value propagated to order. then changing the `C_BPartner_Location.EMail` does not affect the email propagation.